modules/budgets/frontend/module/augment/planned-costs-form.ts

```diff
@@ -26,9 +26,11 @@
 // See COPYRIGHT and LICENSE files for more details.
 //++
 
+import { delegate } from '@knowledgecode/delegate';
+
 export class PlannedCostsFormAugment {
```

Comment:
There is a WIP PR that removes this file: https://github.com/opf/openproject/pull/20774